frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -602,6 +624,29 @@ const MakerForm = ({
 
           <Grid item sx={{ width: '100%' }}>
             <Collapse in={maker.advancedOptions}>
+              <Tooltip
+                enterTouchDelay={0}
+                placement='top-start'
+                title={makerHasAmountRange ? t('Enable Amount Range') : t('Enable Exact Range')}
```

Comment:
Let's run `cd frontend/static/locales; python collect_phrases.py` to collect these new phrases for translation as well.

Reply:
I ran the code, but the json generated wasn't formatted properly, and the order was changed. Was this the expected result, or is it an error in the configuration of my text editor?
![image](https://github.com/RoboSats/robosats/assets/30505180/6678e760-db71-47d6-af69-c37b7299f6f2)

Reply:
This is expected, but pre-commit (prettier) will reformat it before it's committed.

Reply:
Resolved in the last commit f9b6273.